### PR TITLE
disable archive DELETE

### PR DESCRIPTION
**Pull request by mhilton**
Disable deleting of archives until the behaviour for deleting published charms is determined.

### CODE REVIEW

File: internal/v5/archive.go [hunk 1]
```diff
@@ -43,8 +43,7 @@ import (
 func (h *ReqHandler) serveArchive(id *charm.URL, w http.ResponseWriter, req *http.Request) error {
 	resolveId := h.ResolvedIdHandler
 	switch req.Method {
-	case "DELETE":
-		return resolveId(h.AuthIdHandler(h.serveDeleteArchive))(id, w, req)
+	// TODO: support DELETE when it it understood how that interracts with channels.
```
Comment (rogpeppe) [inline internal/v5/archive.go]: s/interracts/interacts/

Comment (frankban) [inline internal/v5/archive.go]: s/it it/it is/ as well

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1633/
Test PASSed.

File: internal/v5/archive.go (hunk 1 above)
Comment (mhilton) [inline internal/v5/archive.go]: done

**Comment by rogpeppe** [comment]
LGTM. Let's add a card to the board to look at it again too.

**Comment by frankban** [comment]
:+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1634/
Test PASSed.

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**